### ISSUE TITLE
[RE2 VR] Movement is not smooth

### ISSUE THREAD

**Issue by FolkSong** [praydog/REFramework#124]
When you walk or run, the first-person camera lurches with each step the character takes. This is not good for motion sickness, it should be completely smooth.

I see there is an option called "FirstPerson_SmoothXZMovementVR" but I don't notice any difference with it on or off.

**Comment by ArmandVestibule** [comment]
I, too, had this issue.  I haven't tested it yet, but there is an ingame option to disable camera wobble in the options.  Does that help?  I won't be able to confirm it for myself until later today.

**Comment by nickyb78** [comment]
Camera wobble option doesnt do anything for me. I think its because the first persin camera is attached to the body so that optiin doesnt work. Iv gotten used to the cameea movement now but it would be good to have an option to get rid of the slight wobble when moving i agree.

**Comment by nickyb78** [comment]
Its still an amazing job by praydog. It feels almost a native vr game like 7 was. :)

**Comment by FolkSong** [comment]
Yes I did try disabling camera wobble in the game options, and also set camera shake to 0 in the mod options.

I agree the mod is fantastic, already one of the best VR mods ever.

**Comment by FolkSong** [comment]
I just found out a legend named Ashok released a separate mod to fix this. Haven't tried it yet but it looks great.

https://www.nexusmods.com/residentevil22019/mods/1022

**Comment by bferguson3** [comment]
I know this is a necro but the mod linked above does not work as of the most recent build.
